gck-rpc-tls-psk.c

```diff
@@ -201,10 +200,29 @@ _tls_psk_client_cb(SSL *ssl, const char *hint,
 		   char *identity, unsigned int max_identity_len,
 		   unsigned char *psk, unsigned int max_psk_len)
 {
-	/* Client tells server which identity it wants to use in ClientKeyExchange */
-	snprintf(identity, max_identity_len, "%s", tls_psk_identity);
```

Comment:
I think it would be better to leave this part and do the parsing in `gck_rpc_init_tls_psk` - basically adding condition before that `snprintf(tls_psk_identity` line that would check if identity is NULL (which currently always is but we could potentially change it in the future) and do that parsing in such case.

Reply:
Fair enough. I've force-pushed this change now.